src/compile/data/bin.ts

```diff
@@ -65,7 +71,6 @@ function parse(model: Model): Dict<VgTransform[]> {
         });
       }
       // FIXME: current merging logic can produce redundant transforms when a field is binned for color and for non-color
```

Comment:
Outdated FIXME -- You are fixing this issue, so you should remove FIXME from the code.